packages/mystmd-py/.gitignore

```diff
@@ -3,3 +3,5 @@ dist
 src/mystmd_py/myst.cjs
 **/__pycache__/
 _package.json
+
+/_build/
```

Comment:
Probably you running myst in this folder?!

Reply:
Lol the one time I just use git add update!